gc.c

```diff
@@ -10699,22 +10699,35 @@ gc_compact(VALUE self)
 
 #if GC_CAN_COMPILE_COMPACTION
 static VALUE
-gc_verify_compaction_references(rb_execution_context_t *ec, VALUE self, VALUE double_heap, VALUE toward_empty)
+gc_verify_compaction_references(rb_execution_context_t *ec, VALUE self, VALUE double_heap, VALUE expand_heap, VALUE toward_empty)
```

Comment:
When changing the arity of this function, also `! GC_CAN_COMPILE_COMPACTION` side.

Reply:
Thank you 👍